html/pfappserver/root/static/admin/config/provisioners.js

```diff
@@ -0,0 +1,113 @@
+$(function() { // DOM ready
+    var items = new Provisioners();
+    var view = new ProvisionerView({ items: items, parent: $('#section') });
+});
+
+/*
+ * The Provisioners class defines the operations available from the controller.
+ */
+var Provisioners = function() {
+};
+
+Provisioners.prototype = new Items();
+
+Provisioners.prototype.id  = '#provisioners';
+
+Provisioners.prototype.formName  = 'modalProvisioner';
+
+Provisioners.prototype.modalId   = '#modalProvisioner';
+
+/*
+ * The ProvisionerView class defines the DOM operations from the Web interface.
+ */
+
+
+var ProvisionerView = function(options) {
+    ItemView.call(this,options);
+    var that = this;
+    this.parent = options.parent;
+    this.items = options.items;
+
+    // Hide the sectype, eap_type fields when 'Open' is selected
+    options.parent.on('change', 'form[name="modalProvisioner"] select[name="security_type"]', this.togglesectype);
+    // Hide the ca_cert_path, cert_type, reversedns and company fields when 'PEAP' is selected
+    options.parent.on('change', 'form[name="modalProvisioner"] select[name="eap_type"]', this.togglecert);
+    // Hide fileds on opening the provisioner
+    options.parent.on('show', '#modalProvisioner', function(e) {
+      that.togglecert(e);
+      that.togglesectype(e);
+    }); 
+};
+
+ProvisionerView.prototype = (function(){
+    function F(){};
+    F.prototype = ItemView.prototype;
+    return new F();
+})();
+
+ProvisionerView.prototype.constructor = ProvisionerView;
+
+ProvisionerView.prototype.togglesectype = function(e) {
```

Comment:
that method is incredibly badly named and cased.

You need to follow the code style you're working in. 

A good name, properly cased would be : toggleSecurityType